CIP-0062/README.md

```diff
@@ -246,59 +247,36 @@ All TxSignErrors defined in [CIP-30](https://cips.cardano.org/cips/cip30/#txsign
 * UserDeclined - Raised when the user declined to sign the entire transaction, in the case of a vote submission, this would be returned if the user declined to sign ALL of the votes.
 * VoteRejected - On a vote transaction, where there may be multiple votes.  If the user accepted some votes, but rejected others, then this error is raised, AND `rejectedVotes` is present in the Error instance.
 
-### Catalyst Extension to CIP-30
-
-#### cardano.{walletName}.catalyst.apiVersion: String
-
-The version number of the Catalyst Extension API that the wallet supports.
-
-#### cardano.{walletName}.catalyst.enable(purpose: VotingPurpose[]): Promise\<API>
-
-Errors: [`APIError`](#extended-apierror)
-
-The `cardano.{walletName}.catalyst.enable()` method is used to enable the
-Catalyst API. It should request permission from the wallet to enable the API for the requested purposes.
-
-If permission is granted, the rest of the API will be available. The wallet
-should maintain a specific whitelist of allowed clients and voting purposes for
-this API. This whitelist can be used to avoid asking for permission every time.
-
-This api, being an extension of
-[CIP-30](https://cips.cardano.org/cips/cip30/),
-expects that `cardano.{walletName}.enable()` to be enabled and added to CIP-30
-whitelist implicitly.
-
-When both this API and [CIP-30](https://cips.cardano.org/cips/cip30/) being
-enabled, is up to the wallet to decide the number of prompts requesting
-permissions to be displayed to the user.
-
-* `purpose` - this is a list of purposes that the dApp is advising that it will be using on the API.  The wallet must respond with an error if the purpose is not supported by the Wallet.
-
-  Note: Currently only voting purpose 0 (Catalyst) is defined. The wallet should reject any other purpose requested.
-
-##### Returns
-
-Upon successful connection via
-[`cardano.{walletName}.catalyst.enable()`](#cardanowalletnamecatalystenablepurpose-votingpurpose-promiseapi),
-a javascript object we will refer to as `API` (type) / `api` (instance) is
-returned to the dApp with the following methods.
+### Catalyst API Extension to CIP-30
 
-### Catalyst API
+The following endpoints extend [CIP-30's Full API](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0030#full-api).
 
 Except `signVotes`, no other method should require any user
```

Comment:
shouldn't also `submitDelegation` be mentioned here?

Reply:
Yup good spot - fixed,